src/clr-angular/data/stack-view/stack-block.ts

```diff
@@ -80,11 +89,26 @@ export class ClrStackBlock implements OnInit {
     }
   }
 
+  /**
+   * Depth of the stack view starting from 1 for first level
+   */
+  @Input('clrStackViewLevel') ariaLevel: number;
```

Comment:
From what I see here and in the demos, this implementation means consumers will have to manage these values and updated them as the model changes. I'm not clear from reading #3568 if thats how this should behave. 

imo - The stackblock component should handle these values internally so that the consumer doesn't need to worry about managing these values.

Reply:
per discussion below this isn't relevent.